dotCMS/src/main/webapp/html/portlet/ext/contentlet/edit_contentlet.jsp

```diff
@@ -51,6 +51,7 @@
 	if (isNewBinaryFieldEnabled != null && isNewBinaryFieldEnabled.equalsIgnoreCase("true")) {
 %>
 	<script src="/html/contenttype-fields.js"></script>
+	<link rel="stylesheet" href="/html/css/contenttype-fields-styles.css" />
```

Comment:
Should we rename this whole thing to "binary-field.js" and "binary-field.scss"? 🤔

Reply:
Sure, we can do it!